wasm/src/main/java/com/dylibso/chicory/wasm/RawParserListener.java

```diff
@@ -0,0 +1,9 @@
+package com.dylibso.chicory.wasm;
+
+import com.dylibso.chicory.wasm.types.RawSection;
+
+@FunctionalInterface
+public interface RawParserListener {
```

Comment:
Genuine question, we use the same patter with a Listener for normal Parsing, what's the advantage we get against a more direct approach?

Reply:
It seemed a bit cleaner to avoid the cast in the listener implementation. It's more type-safe, in that the method is guaranteed to always have `RawSection`.

But I can remove this as we only use it one place.